docs/settings/reporting-settings.asciidoc

```diff
@@ -203,26 +203,35 @@ The `file:` protocol is always denied, even if no network policy is configured.
 ==== CSV settings
 
 [[xpack-reporting-csv]] `xpack.reporting.csv.maxSizeBytes` {ess-icon}::
-The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `10mb`.
+The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `250mb`.
 
 [NOTE]
 ============
-Setting `xpack.reporting.csv.maxSizeBytes` much larger than the default 10 MB limit has the potential to negatively affect the
-performance of {kib} and your {es} cluster. There is no enforced maximum for this setting, but a reasonable maximum value depends
-on multiple factors:
+We recommend using CSV reports to export moderate amounts of data only. The feature enables analysis of data in external tools, but is not designed for bulk export or to backup {es} data. Setting `xpack.reporting.csv.maxSizeBytes` larger than the default limit hat the potential to negatively affect the performance of your {es} cluster. 
 
-* The `http.max_content_length` setting in {es}.
-* Network proxies, which are often configured by default to block large requests with a 413 error.
-* The amount of memory available to the {kib} server, which limits the size of CSV data that must be held temporarily.
+The following deployment configurations may lead to failed report jobs or incomplete reports:
 
-For information about {kib} memory limits, see <<production, using {kib} in a production environment>>.
+* Any shard needed for search is unavailable.
+* Data is stored on slow storage tiers. 
+* Network latency between nodes is high.
+* Cross-cluster search is used. 
+
+To export large amounts of data we recommend using {es} APIs directly, or Logstash CSV export. See {ref}/scroll-api.html[Scroll API], {ref}/point-in-time-api.html[Point in time API], and {ref}/sql-rest-format.html#_csv[CSV response format].
 ============
 
 `xpack.reporting.csv.scroll.size`::
 Number of documents retrieved from {es} for each scroll iteration during a CSV export. Defaults to `500`.
+[NOTE]
+============
+If the default size of 500 covers a large number of shards in {es}, you may need to lower this setting. 
+============
 
 `xpack.reporting.csv.scroll.duration`::
  Amount of {time-units}[time] allowed before {kib} cleans the scroll context during a CSV export. Defaults to `30s`.
+[NOTE]
+============
+If search latency in {es} is sufficiently high, such as if you are using cross-cluster search, you may need to increase the setting. 
```

Comment:
```suggestion
If search latency in {es} is sufficiently high, such as if you are using {ccs}, you may need to increase the setting. 
```

Reply:
Other than cross-cluster search, using frozen tiers is another example where an admin may need to increase this setting.